packages/teams-js/src/internal/communication.ts

```diff
@@ -169,6 +227,22 @@ export function sendAndHandleSdkError<T>(actionName: string, ...args: any[]): Pr
   });
 }
 
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function sendMessageToParentAsyncWithVersion<T>(
+  apiVersion: string,
```

Comment:
It would be good to do some validation here that the version is being passed in a consistent format: not empty or null/undefined, starts with a v, lower case, etc. Want to prevent someone who isn't you from adding a change in the future that passes in an apiVersion that is differently formatted which could cause problems downstream.

Reply:
I added a check function in run-time but I don't think it is in best practice. Let's start from this point and discuss how to improve that.